<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1071</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="seattleButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250,250,250);
border:none;
background-color: rgb(51, 167, 255);

</string>
    </property>
    <property name="text">
     <string>SEATTLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="japanButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250,250,250);
border:none;
background-color: rgb(51, 167, 255);</string>
    </property>
    <property name="text">
     <string>JAPAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="londonButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250,250,250);
border:none;
background-color: rgb(51, 167, 255);</string>
    </property>
    <property name="text">
     <string>LONDON</string>
    </property>
   </widget>
   <widget class="QLabel" name="tempLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>weather</string>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>condi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>icon</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="blurBackground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(103, 103, 103);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255, 255);    

border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>date</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hourLcd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255, 255); 
	border-color:rgb(51, 167, 255);   
border-width : 1.2px;
border-style:inset;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLcd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255, 255); 
	border-color:rgb(51, 167, 255);   
border-width : 1.2px;
border-style:inset;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondLcd">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255, 255); 
	border-color:rgb(51, 167, 255);   
border-width : 1.2px;
border-style:inset;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background:transparent;
color:rgb(255, 255, 255);

}

</string>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="locationButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250,250,250);
border:none;
background-color: rgb(51, 167, 255);
</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>1041</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(118, 118, 118);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tempOneLabel">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>550</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>temp1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>condi1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tempLabel2">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>temp2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>condi2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tempLabel3">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>550</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>temp3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>comdi3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255); </string>
    </property>
    <property name="text">
     <string>Welcome message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="blurMapBackground">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>351</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(120, 120, 120);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>331</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="zipcodeBlur">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(120, 120, 120);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clearMapButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250,250,250);
border:none;
background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/clear1.png</normaloff>:/images/clear1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="threeHourLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>icon1</string>
    </property>
   </widget>
   <widget class="QLabel" name="sixHourLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>icon2</string>
    </property>
   </widget>
   <widget class="QLabel" name="nineHourLabel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>icon3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255, 255); </string>
    </property>
    <property name="text">
     <string>To-Do List</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView
{
color:rgb(255, 255, 255);
background-color: transparent;
border:none;

}</string>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addToDoListButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>+3hrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>+6hrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>510</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: rgb(255,255, 255);    </string>
    </property>
    <property name="text">
     <string>+9hrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="emptyLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255, 255); </string>
    </property>
    <property name="text">
     <string>(Empty)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>backgroundLabel</zorder>
   <zorder>zipcodeBlur</zorder>
   <zorder>testLabel</zorder>
   <zorder>blurBackground</zorder>
   <zorder>seattleButton</zorder>
   <zorder>japanButton</zorder>
   <zorder>londonButton</zorder>
   <zorder>tempLabel</zorder>
   <zorder>conditionLabel</zorder>
   <zorder>iconLabel</zorder>
   <zorder>dateLabel</zorder>
   <zorder>hourLcd</zorder>
   <zorder>minuteLcd</zorder>
   <zorder>label</zorder>
   <zorder>secondLcd</zorder>
   <zorder>zipCodeEdit</zorder>
   <zorder>locationButton</zorder>
   <zorder>tempOneLabel</zorder>
   <zorder>conditionLabel1</zorder>
   <zorder>tempLabel2</zorder>
   <zorder>conditionLabel2</zorder>
   <zorder>tempLabel3</zorder>
   <zorder>conditionLabel3</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>welcomeLabel</zorder>
   <zorder>blurMapBackground</zorder>
   <zorder>mapLabel</zorder>
   <zorder>clearMapButton</zorder>
   <zorder>threeHourLabel</zorder>
   <zorder>sixHourLabel</zorder>
   <zorder>nineHourLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>tableView</zorder>
   <zorder>line_4</zorder>
   <zorder>addToDoListButton</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>emptyLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
